chainer/optimizers/adam.py

```diff
@@ -243,6 +327,9 @@ class Adam(optimizer.GradientMethod):
         eta (float): Schedule multiplier, can be used for warm restarts.
         weight_decay_rate (float): Weight decay rate.
         amsgrad (bool): Whether to use AMSGrad variant of Adam.
+        adabound (bool): Whether to use the AdaBound variant of Adam.
+        final_lr (float): final (SGD) learning rate in adabound.
```

Comment:
nit-picking: Same as `AdamRule` (L93). 
How about using a capital letter for consistency with the other arguments?
```suggestion
        final_lr (float): Final (SGD) learning rate in AdaBound.
```